app/src/main/java/org/p2p/wallet/auth/analytics/AuthAnalytics.kt

```diff
@@ -0,0 +1,56 @@
+package org.p2p.wallet.auth.analytics
+
+import org.p2p.wallet.common.analytics.TrackerContract
+
+class AuthAnalytics(private val tracker: TrackerContract) {
+
+    fun logAuthViewed(screenName: String, authType: AuthType) {
+        tracker.logEvent(
+            "Auth_Viewed",
+            arrayOf(
+                Pair("Auth_Source", screenName),
+                Pair("Auth_Type", authType.title)
+            )
+        )
+    }
+
+    fun logAuthValidated(result: AuthResult, authType: AuthType) {
+        tracker.logEvent(
+            "Auth_Validated",
+            arrayOf(
+                Pair("Auth_Result", result.title),
+                Pair("Auth_Type", authType.title)
+            )
+        )
+    }
+
+    fun logAuthResetInvoked() {
+        tracker.logEvent("Auth_Reset_Invoked")
+    }
+
+    fun logAuthResetValidated(result: ResetResult) {
+        tracker.logEvent(
+            "Auth_Reset_Validated",
+            arrayOf(
+                Pair("Auth_Reset_Result", result.title)
```

Comment:
```suggestion
               "Auth_Reset_Result" to result.title
```

А нельзя `Auth_Reset_Result` вынести тоже в enumку?